CONTRIBUTING.md

```diff
@@ -6,9 +6,10 @@ Thank you for your interest in contributing to snarkVM! Below you can find some
 
 Please follow the instructions below when filing a pull request:
 
-- ensure that your branch is forked from the current [master](https://github.com/ProvableHQ/snarkVM/tree/master) branch
-- run `cargo fmt` before you commit; we use the `nightly` version of `rustfmt` to format the code, so you'll need to have the `nightly` toolchain installed on your machine; there's a [git hook](https://git-scm.com/docs/githooks) that ensures proper formatting before any commits can be made, and [`.rustfmt.toml`](https://github.com/ProvableHQ/snarkVM/blob/master/.rustfmt.toml) specifies some of the formatting conventions
-- run `cargo clippy --all-targets --all-features` to ensure that popular correctness and performance pitfalls are avoided
+- Ensure that your branch is forked from the current [master](https://github.com/ProvableHQ/snarkVM/tree/master) branch.
+- Use the git commit hook, which ensures `cargo fmt` and `cargo clippy` are run. [`.rustfmt.toml`](https://github.com/ProvableHQ/snarkVM/blob/master/.rustfmt.toml) specifies some of the formatting conventions.
```

Comment:
if we no longer need to use `nightly` to run `rustfmt`, `.rusty-hook.toml` should be adjusted as well

Reply:
We still use these options in `.rustfmt.toml`, so I don't think we can switch to stable quite yet.

Reply:
Slight change in wording to clarify readers should be looking at the git hook for what command to run: [5ceecd5](https://github.com/ProvableHQ/snarkVM/pull/2651/commits/5ceecd51e792a1ac208db3fffb03f5773c22dd4f)